bin.js

```diff
@@ -662,6 +662,12 @@ async function uploadData (uploadURL, authToken, filename, opts) {
 
   console.log(`Uploading data for ${filePrefix} and ${filePrefix}.html`)
 
+  // Check if path exists before uploading
+  if(!fs.existsSync(path.resolve(filePrefix))) {
+    console.error('File '+path.resolve(filePrefix)+' does not exist')
+    process.exit(1)
```

Comment:
I don't think we should exit early here, but rather throw inside this function instead.

Reply:
@mcollina - I will probably get rid of this code block entirely as @jasnell pointed out the file may not exist when existsSync is called here.